projects/sunbird-video-player/src/lib/sunbird-video-player.component.ts

```diff
@@ -5,6 +5,7 @@ import {
 import { ErrorService , errorCode , errorMessage, ISideBarEvent } from '@project-sunbird/sunbird-player-sdk-v9';
 
 import { PlayerConfig } from './playerInterfaces';
+import { Transcripts } from './playerInterfaces';
```

Comment:
move this logic to ViewerService & pass it as part 'transcripts' as part of PlayerConfig.config itself.